runtime/vscode/overrides/vscode-file-commands.vim

```diff
@@ -49,10 +49,11 @@ AlterCommand e[dit] Edit
 AlterCommand ex Ex
 AlterCommand ene[w] Enew
 AlterCommand fin[d] Find
-AlterCommand w[rite] Write
 AlterCommand sav[eas] Saveas
 AlterCommand wa[ll] Wall
```

Comment:
In response to this comment https://github.com/vscode-neovim/vscode-neovim/pull/2027#issuecomment-2141140685

I still need to add a check to make sure that `wall` doesn't affect lots of other buffers

Reply:
Actually, looks like this check is unnecessary. With our current `wall` implementation, BufWriteCmd is not fired